Wangscape/tilegen/alpha/CalculatorTopTwo.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <utility>
+#include <vector>
+
+#include "CalculatorBase.h"
+
+
+namespace tilegen
+{
+namespace alpha
+{
+
+
+class CalculatorTopTwo :
+    public tilegen::alpha::CalculatorBase
+{
+public:
+    double power = 1.;
+protected:
+    virtual void updateAlphasAux(const Weights& weights);
```

Comment:
You're right, some of the features of #37 will probably need the `CalculatorBase` class to change. Using `override` in classes derived from `CalculatorBase` could help to detect errors when fully implementing #37.